### ISSUE TITLE
Exposing notification icon settings

### ISSUE THREAD

**Issue by cboar** [flurry/react-native-flurry-sdk#7]
Hi, I'm facing an issue with the push notification icon being a white square on Android. It seems [the FlurryMarketingOptions aren't configurable from the FlurryModule.Builder.](https://github.com/flurry/react-native-flurry-sdk/blob/b0afff43bc45da25d0a1899534c10221009fe60d/android/src/main/java/com/flurry/android/reactnative/FlurryModule.java#L380) It seems this setting was the only solution to fix the square icon. Would it be possible to expose this in some form? Thanks in advance!

**Comment by flurrydev** [comment]
Thanks for the feedback!
For the next release, `FlurryModule.Builder` will accept user's messaging options or listener as,
```
- .withMessaging(true)
- .withMessaging(true, myFlurryMarketingOptions)
- .withMessaging(true, myFlurryMessagingListener)
```
When user provides its own options or listener, the JavaScript listener will be disabled.

**Comment by poting-oath** [comment]
Thanks for the feedback!
For the next release, `FlurryModule.Builder` will accept user's messaging options or listener as,
```
- .withMessaging(true)
- .withMessaging(true, myFlurryMarketingOptions)
- .withMessaging(true, myFlurryMessagingListener)
```
When users provide their own options or listener, the JavaScript listener will be disabled.

**Comment by poting-oath** [comment]
@cboar We just released a new version 3.2.0 with the feature that allows optional user's native `FlurryMarketingOptions` or `FlurryMessagingListener`. Thanks a lot for your great feedback!
